capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -27,63 +42,73 @@ class AdminClubPage extends Component {
     let data = {};
     const formElements = document.getElementById("update-club-form").elements;
     for (let i = 0; i < formElements.length; i++) {
-      if (formElements[i].name.length !== 0 && 
-          formElements[i].type !== "submit" && formElements[i].value.length !== 0) {
+      if (formElements[i].name.length !== 0 &&
+        formElements[i].type !== "submit" && formElements[i].value.length !== 0) {
```

Comment:
nit: Good catch on removing the trailing whitespace in the first line. The indentation on the second line should be reverted though to follow the [rectangle rule](https://github.com/google/google-java-format/wiki/The-Rectangle-Rule).

Reply:
Will change, seems the gitpod format document is a little finicky.